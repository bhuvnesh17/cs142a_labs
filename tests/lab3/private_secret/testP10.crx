// methods by the same name

func gorn(a:int) : int {
  return a;
}

func main() : void {
  return ::main();
}

func gorn(b:float) : float {
  return b;
}
