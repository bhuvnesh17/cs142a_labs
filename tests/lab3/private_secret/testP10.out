Error parsing file.
DeclareSymbolError(11,6)[gorn already exists.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(gorn)
Symbol(main)


